### PR TITLE
Yelp API Spout

### PR DESCRIPTION
**Pull request by rj1061**
This is a working Spout for the Yelp API. This fetches everything that we need to populate the new york restaurant directory. Doesn't emit yet since we might want to format here. Returns JSON State.

### CODE REVIEW

File: src/spouts/yelp.py [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+        self.logger.info("Log: " + r.content)
+        self.access_json = json.loads(r.text)
+        self.token = self.access_json['access_token']
+        self.searchJson = requests.request('GET',
+                                           'https://api.yelp.com/v3/businesses/search',
+                                           params = {'location': 'New York',
+                                                   'term': 'restaurants'},
+                                           headers = {'Authorization': 'Bearer ' + self.token})
+        self.logger.info("Respose: " + self.searchJson.content)
         self.emit(['something'])
```
Comment (rj1061) [inline src/spouts/yelp.py:24]: We can change  the emit object to be in the format most easy to store.

**Review by maqzi — APPROVED** [review]
im good

**Comment by muraleo** [comment]
I'm good

File: src/spouts/yelp.py (hunk 1 above)
Comment (psk285) [reply src/spouts/yelp.py:24]: I'm good

**Pull request merged**